airgun/entities/subscription.py

```diff
@@ -103,6 +103,24 @@ def read_delete_manifest_message(self):
             manage_view.close_button.click()
         return delete_message
 
+    def read_expire_manifest_message(self):
+        """Read message displayed about 'manifest expiration' at Subscription Manifest section"""
+        expire_manifest_header = None
+        expire_manifest_message = None
+        view = self.navigate_to(self, 'Manage Manifest')
+        view.wait_animation_end()
+        if view.manifest_alert.is_displayed:
+            expire_manifest_header = view.manifest_expire_header.read()
+            expire_manifest_message = view.manifest_expire_message.read()
+        else:
+            # Subscription Manifest header & message is not present
+            expire_manifest_message = 'Manifest expire alert not found'
+        # close opened modal dialogs views
+        manage_view = ManageManifestView(self.browser)
+        if manage_view.is_displayed:
+            manage_view.close_button.click()
+        return expire_manifest_header, expire_manifest_message
```

Comment:
A small suggestion - wouldn't it better to return a dict here
```suggestion
        return {'header'=expire_manifest_header, 'message'=expire_manifest_message}
```
and reference it via keywords instead of indexes in the test? It would improve the test readability IMHO.

Reply:
sure, nice suggestion.